components/shared/InfoTooltip.tsx

```diff
@@ -1,14 +1,18 @@
-import { useState } from "react";
+import { useState, useCallback } from "react";
```

Comment:
`useCallback` is no longer being used, so this import can be removed.

```suggestion
import { useState } from "react";
```